### DISCUSSION TITLE
Git info has disappeared from prompt

### DISCUSSION THREAD

**Discussion by slackline** [ohmyzsh/ohmyzsh#12215] [Help] [answered]
I use the [alien theme](https://github.com/eendroroy/alien) and share my config via dotfiles across multiple systems.

On two of them the git information has stopped showing without any apparent change in the configuration.

I set the theme from `~/.zshrc` with...

```
if [ "${HOST}" = "host1" ] || [ "${HOSTNAME}" = "host1" ]; then
    export ALIEN_THEME="soft"
elif [ "${HOST}" = "host2" ] || [ "${HOSTNAME}" = "host2" ]; then
    export ALIEN_THEME="blue"
elif [ "${HOST}" = "host3" ] || [ "$HOSTNAME" = "host3" ]; then
    export ALIEN_THEME="green"
elif [ "${HOST}" = "host4" ] || [ "${HOSTNAME}" = "host4" ]; then
    export EMACS_SOCKET="/run/user/1000/emacs/server"
    export ALIEN_THEME="blue"
elif [ "${HOST}" = "host5" ] || [ "${HOSTNAME}" = "host5" ]; then
    export EMACS_SOCKET="/run/user/1000/emacs/server"
    export ALIEN_THEME="gruvbox"
fi

ZSH_THEME="alien/alien"
```

The `$PROMPT` now differs between systems. On one where the git info is still shown it is...

```
%F{232}%K{%(?.226.196)}%(?.. %? )%k%f%F{%(?.226.196)}%K{22}%k%f%F{254}%K{22} 8:02:08 am UTC %k%f%F{22}%K{28}%k%f%F{254}%K{28} 0 %k%f%F{28}%K{34}%k%f%F{232}%K{34} user1 %k%f%F{34}%K{82}%k%f%F{232}%K{82} %~ %k%f%F{82}%K{238}%k%f%F{154}%K{238} G  master %k%f%F{238}%K{244}%k%f%F{255}%K{244} @ %k%f%F{244}%K{253}%k%f%F{16}%K{253} %F{65}+%f %F{236}+%f %F{65}⭑%f %F{236}⭑%f %F{65}-%f %F{236}-%f %F{236}?%f %k%f%F{253}%K{235}%k%f%F{228}%K{235} %k%f%F{235}%f
%F{245}192.168.1.45 %f%F{228}%B❱%b %f
```

On a system where it has disappeared it reads...

```
%F{0}%K{%(?.87.202)}%(?.. %? )%k%f%F{%(?.87.202)}%K{229}%k%f%F{0}%K{229} 8:13:32 am GMT %k%f%F{229}%K{193}%k%f%F{0}%K{193} 0 %k%f%F{193}%K{87}%k%f%F{0}%K{87} user1 %k%f%F{87}%K{45}%k%f%F{0}%K{45} %~ %k%f%F{45}%K{241}%k%f%F{87}%K{241} %k%f%F{241}%f
%F{238}192.168.1.1 %f%F{87}%B❱%b %f
```

I've run `upgrade_oh_my_zsh` / `omz upgrade` on both systems and the `~/.oh-my-zsh/` repositories are both at `HEAD` on commit `f9bbf604`.

I've gone into `~/.oh-my-zsh/custom/theme/alien` and made sure its up-to-date.

If I check the value of `$ALIEN_SECTIONS_LEFT` it includes `vcs_branch:async` `vcs_status:async`  and `vcs_dirty:async` 

```
 echo $ALIEN_SECTIONS_LEFT 
exit time battery user path vcs_branch:async vcs_status:async vcs_dirty:async versions:async newline ssh venv prompt
```

I've tried with a couple of other included themes such as `robbyrussell` and `maran` and the git status is shown in those prompts so I'm unsure why they're not showing for the Alien theme (which looking at the git history hasn't been updated in some time), so I'm unsure why they're not showing. Perhaps something has changed in the `oh-my-zsh` framework that needs updating?

Any suggestions on how to recover this would be very much appreciated, I like the Alien theme because of the ability to have different coloured themes on a per host basis as it allows me to quickly see without having to read the prompt which host I'm on.

**Comment by mcornella** [comment, marked as answer]
The alien theme is not included in OMZ, please seek support on their GitHub
project.

**Reply by slackline** [reply]
Thanks for the pointer, I have done that too, was just curious if there was anything on the OMZ side that might have changed as the alien theme doesn't appear to have changed for quite some time.
